src/cairoWriter/writers/parameterListWriter.ts

```diff
@@ -30,14 +30,32 @@ export class ParameterListWriter extends CairoASTNodeWriter {
         this.ast,
         varTypeConversionContext,
       );
-      if (tp instanceof CairoDynArray && node.parent instanceof FunctionDefinition) {
-        return isExternallyVisible(node.parent) ||
-          node.getClosestParentByType(ContractDefinition)?.name.includes('@interface')
-          ? `${value.name}_len : ${tp.vLen.toString()}, ${value.name} : ${tp.vPtr.toString()}`
-          : `${value.name} : ${tp.toString()}`;
+      if (isReturnParamList(node)) {
```

Comment:
not necessarily need to define a new function for this, we'd just insert the check statement inside the `if`

Reply:
It is not clear what the condition do per se, so I thought could be a good idea to wrap it in a more meaningful representation